api/src/main/java/org/openmrs/api/db/PatientDAO.java

```diff
@@ -296,5 +297,11 @@
 	 * @return the saved allergy
 	 */
 	public Allergy saveAllergy(Allergy allergy);
+
+	/**
```

Comment:
It would be great to add a description to this method.

Reply:
@dkayiwa done